### PR TITLE
feat: update webhook verify docs

### PR DESCRIPTION
**Pull request by sebi75**
image

image

image

### CODE REVIEW

File: api-reference/webhooks.mdx [hunk 1]
```diff
@@ -62,54 +62,55 @@ the signature is made out of 2 components:
...
-  const signature = req.headers['Sync-Signature'];
-  const payload = req.body.toString(); // Raw payload for validation
-
-    if (!verifySignature(payload, signature, WEBHOOK_SECRET)) {
-      return res.status(400).send('Invalid signature');
-    }
-
-    // If the signature is valid, process the webhook
-});
+app.post("/webhook", (req: any, res: any) => {
```
Comment (assimovt) [inline api-reference/webhooks.mdx]: @sebi75 lets remove type annotations

File: api-reference/webhooks.mdx [hunk 2]
```diff
@@ -62,54 +62,55 @@ the signature is made out of 2 components:
 <CodeGroup>
 
 ```javascript javacript
```
Comment (assimovt) [inline api-reference/webhooks.mdx]: do u need second javascript def? probly first should do it

File: api-reference/webhooks.mdx [hunk 3]
```diff
@@ -62,54 +62,55 @@ the signature is made out of 2 components:
...
-  const payload = req.body.toString(); // Raw payload for validation
-
-    if (!verifySignature(payload, signature, WEBHOOK_SECRET)) {
-      return res.status(400).send('Invalid signature');
-    }
-
-    // If the signature is valid, process the webhook
-});
+app.post("/webhook", (req: any, res: any) => {
+  const signature = req.headers["sync-signature"];
```
Comment (assimovt) [inline api-reference/webhooks.mdx:100]: nit: line break

File: api-reference/webhooks.mdx [hunk 4]
```diff
@@ -62,54 +62,55 @@ the signature is made out of 2 components:
...
+  if (!verifySignature(req.body, signature, WEBHOOK_SECRET)) {
+    return res.status(400).json({
+      message: "Invalid signature",
+    });
+  }
 
-app.listen(3000, () => {
-  console.log('Webhook server listening on port 3000');
+  console.log("Webhook event received with valid signature");
```
Comment (assimovt) [inline api-reference/webhooks.mdx]: remove console.log

File: api-reference/webhooks.mdx [hunk 5]
```diff
@@ -118,40 +119,54 @@ import hashlib
...
 
-    if not verify_signature(payload, signature, WEBHOOK_SECRET):
-      return Response('Invalid signature', status=400)
-            
-    # If signature is valid, process the webhook
-    return Response('Webhook received', status=200)
+    if not verify_signature(request.get_json(), signature, WEBHOOK_SECRET):
+        return jsonify({'message': 'Invalid signature'}), 400
+
+    print("Webhook event received with valid signature")
```
Comment (assimovt) [inline api-reference/webhooks.mdx]: remove console, response code is ok

**Review by assimovt — APPROVED** [review]
minor nits, otherwise lgtm

File: api-reference/webhooks.mdx (hunk 2 above)
Comment (sebi75) [reply api-reference/webhooks.mdx]: yes- first one is syntax highlighting, second is mintlify tab def

**Pull request merged**